keps/sig-scheduling/3243-respect-pod-topology-spread-after-rolling-upgrades/README.md

```diff
@@ -837,6 +898,7 @@ Think about adding additional work or introducing new steps in between
 
 [existing SLIs/SLOs]: https://git.k8s.io/community/sig-scalability/slos/slos.md#kubernetes-slisslos
 -->
+No.
```

Comment:
The impact should be fairly small, but it actually impact scheduling latency, which directly contributes to pod-startup-latency SLO.

Reply:
Updated